To-do list/Controllers/TaskDetailsViewController.swift

```diff
@@ -18,18 +18,33 @@ class TaskDetailsViewController: UIViewController {
     @IBOutlet private weak var titleTextField: UITextField!
     @IBOutlet private weak var descriptionTextField: UITextField!
     @IBOutlet private weak var statusButton: UIButton!
+    @IBOutlet private weak var dateTextField: UITextField!
     
     var task = Task()
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        
         prepareUIElements()
+        
+        let datePicker = UIDatePicker()
```

Comment:
We can move this code to some function, similar to prepareUIElements()

Reply:
done